src/Model/batchprocessing/batchprocessingMachineLearning/Preprocessing.py

```diff
@@ -0,0 +1,461 @@
+import pandas as pd
+# Skit Learn Modules
+from sklearn.model_selection import train_test_split
+from sklearn.preprocessing import StandardScaler  # standartiztion
+from sklearn.preprocessing import OneHotEncoder
+
+from sklearn.compose import ColumnTransformer
+from sklearn.utils import resample
+from collections import Counter
+import logging
+import ast
+
+pd.options.mode.chained_assignment = None  # default='warn'
+
+"""
```

Comment:
https://www.programiz.com/python-programming/docstrings

Reply:
Thank you.
I added docstring to all functions